posthog/settings/web.py

```diff
@@ -421,3 +421,5 @@ def static_varies_origin(headers, path, url):
 HOG_TRANSFORMATIONS_CUSTOM_HOG_ENABLED = get_from_env(
```

Comment:
Personally I would just get rid of this value as its a duplicate of the other one.